spec/acceptance/puppetserver_upgrade_spec.rb

```diff
@@ -1,6 +1,8 @@
 require 'spec_helper_acceptance'
 
-unless unsupported_puppetserver
+unless unsupported_puppetserver ||
+    (fact('os.family') == 'RedHat' && fact('os.release.major') == '9') ||
+    (fact('os.name') == 'Ubuntu' && fact('os.release.major') == '22.04')
```

Comment:
or should I move this to a `unsupported_puppetserver_upgrade` helper?

Reply:
I was also thinking about that.